### PR TITLE
SERENITY-1295 Update url creation

### PR DESCRIPTION
**Pull request by mtzavellas-cw**

### CODE REVIEW

File: src/python/serenity_sdk/client.py [hunk 1]
```diff
@@ -16,19 +16,21 @@ class Environment(Enum):
     """
     DEV = 'dev'
     TEST = 'test'
-    PRODUCTION = 'prod'
+    PRODUCTION = ''
```
Comment (kdowney-cloudwallcapital) [inline src/python/serenity_sdk/client.py:19]: why is the label blank, @mtzavellas-cw? Not sure I like that; better to be explicit.

File: src/python/serenity_sdk/client.py [hunk 2]
```diff
@@ -16,19 +16,21 @@ class Environment(Enum):
...
 
 
 class Region(Enum):
     """
     The regional installation of Serenity to use for connection purposes.
     """
-    US_PRIMARY = 'eastus2'
+    GLOBAL = ''
```
Comment (kdowney-cloudwallcapital) [inline src/python/serenity_sdk/client.py:26]: same comment here, should be "global" in my view.

File: src/python/serenity_sdk/client.py [hunk 3]
```diff
@@ -82,11 +85,23 @@ def load_local_config(config_id: str, config_dir: str = None) -> Any:
...
 
-def get_scopes(env: Environment = Environment.PRODUCTION, region: Region = Region.US_PRIMARY) -> List[str]:
+def get_scopes(env: Environment = Environment.PRODUCTION, region: Region = Region.GLOBAL) -> List[str]:
     """
     Helper function that returns the login scopes required to access the API given an environment
     and a region. In general you do not need to call this directly.
     """
     return [
-        f'https://serenity-api-{env.value}-{region.value}.cloudwall.network/.default'
+        f"{_get_url('https://serenity-api', env, region)}/.default"
```
Comment (kdowney-cloudwallcapital) [inline src/python/serenity_sdk/client.py:94]: ah, OK, I see, disregard my comments.

**Review by kdowney-cloudwallcapital — APPROVED** [review]

File: src/python/serenity_sdk/client.py (hunk 3 above)
Comment (mtzavellas-cw) [reply src/python/serenity_sdk/client.py:94]: Yes, I was thinking of adding a new enum value called `EMPTY` but it looks confusing if we use it as parameter inputs like
`client = SerenityClient(config, Environment.EMPTY, Region.EMPTY)`.
It looks better as
`client = SerenityClient(config, Environment.PRODUCTION, Region.GLOBAL)`.

Another approach is that we check explicitly if the Environment is PROD and Region is GLOBAL and set it as empty when generating the URL. But that looks ugly too.

Happy to take a better approach.

**Pull request merged**